src/transformers/models/resnet/modeling_resnet.py

```diff
@@ -314,7 +308,7 @@ def __init__(self, config):
         self.config = config
         self.embedder = ResNetEmbeddings(config.num_channels, config.embedding_size, config.hidden_act)
         self.encoder = ResNetEncoder(config)
-        self.pooler = nn.AdaptiveAvgPool2d((1, 1))
+        self.poolinger = nn.AdaptiveAvgPool2d((1, 1))
```

Comment:
`poolinger` is not a word AFAICT, so I don't think this change is welcome. `pooling_layer` if you want something else, but `pooler` is perfectly understandable for me, and we use it in all model outputs (`pooler_output`).

Reply:
Oh I think I search and replace `self.pool` -> `self.pooling` but I have a `self.pooler`